_pages/schedule.md

```diff
@@ -22,3 +22,17 @@ The conference will take place from 9AM to 6PM.
 
 <a class="button" href="{{ site.schedule_link }}" target="_blank">View schedule!</a>
 
+## Sprints
+
+So you have listened to all the exciting topics on Django for three days and now feel motivated do something about it. What could be a better motivation than contributing to the very open source software you just listened to?
+
+On Thursday and Friday, you have the opportunity to do exactly that. This is your opportunity to contribute to Open Source software. Even if you haven't contributed before, this is also an option to take your first steps. In the sprint, you can contribute to Django as you see fit, and if you need help, there will be a pool of experienced contributors to guide you. You can also reach out through slack in the sprints channel.
+
+<!--
+Thursday - 9AM to 6PM
+Friday - 9AM to 6PM
+-->
+
+The Sprints are free for all conference participants. However, as the Sprints venue does not have unlimited capacity, and we would like to prepare a precise amount of food and drinks, please keep your participation status updated on the ticketing website (after buying your ticket).
```

Comment:
```suggestion
The Sprints are free for all conference participants. However, in order for us to ensure we have catered for appropriate numbers, please keep your participation status updated on the ticketing website (after buying your ticket).
```
Made this a bit vaguer; we aren't providing drinks, and although the venue doesn't have unlimited capacity, I don't tknow what we do if we have more attendees than we've assumed (but it's very unlikely so I'm not worried :))